packages/teams-js/src/public/pages.ts

```diff
@@ -750,4 +750,94 @@ export namespace pages {
       return runtime.supports.pages ? (runtime.supports.pages.appButton ? true : false) : false;
     }
   }
+
+  /**
+   * Parameters for the NavigatewithinApp API
+   *
+   * @beta
+   */
+  export interface NavigateWithinAppParams {
+    /**
+     * @deprecated
+     * As of 2.0.0, please use {@link app.PageInfo.id | app.Context.page.id} instead
+     *
+     * Developer-defined ID of the Page to navigate to within the application (Formerly EntityID)
+     */
+    pageId: string;
+
+    /**
+     * @deprecated
+     * As of 2.0.0, please use {@link app.PageInfo.subPageId | app.Context.page.subPageId} instead
+     *
+     * Optional developer-defined ID describing the content to navigate to within the Page. This will be passed
+     * back to the application via the Context object.
+     */
+    subPageId?: string;
+  }
+
+  /**
+   * Provides functions for navigating without your own application ID.
```

Comment:
Maybe "Provides functions for navigating within your application without specifying your applicationId"

Reply:
How about that, but with this tweak: without having to specify your applicationId

Reply:
Resolved in latest commit